### DISCUSSION TITLE
Headers in description have weird margins

### DISCUSSION THREAD

**Discussion by adjsky** [scalar/scalar#1141] [Q&A] [answered]
Hi! I noticed that headings have weird top margins in description.

I used [redoc petstore](https://redocly.github.io/redoc/) as a reference.
![image](https://github.com/scalar/scalar/assets/49305219/cf0a5784-c8ab-4234-b5ad-e70b55dc291e)

I managed to "fix" the issue with the following css hack (i noticed that each header resets its top margin to 0):

```cs
.markdown > :first-child {
  margin-top: revert !important;
}
```

Is this intentional behavior or not?

**Comment by hwkr** [comment]
Hey @adjsky, that doesn't look right, they should probably have some margin.

**Reply by cameronrohani** [reply, marked as answer]
Let's make sure all heading have 

margin-top: 24px

Let's fix this today :)
